From: Leigh Scott <leigh123linux@gmail.com>
Date: Wed, 14 Sep 2022 22:36:40 +0100
Subject: [PATCH] Specify FFmpeg suffix


diff --git a/cmake_modules/Findffmpeg.cmake b/cmake_modules/Findffmpeg.cmake
index 2e813d9a9..4f929da16 100644
--- a/cmake_modules/Findffmpeg.cmake
+++ b/cmake_modules/Findffmpeg.cmake
@@ -73,6 +73,8 @@ function (_ffmpeg_find component headername)
       /opt/lib
       /usr/freeware/lib64
       "${ffmpeg_ROOT}/bin"
+    PATH_SUFFIXES
+      ffmpeg
     DOC "FFMPEG's ${component} library")
   mark_as_advanced("ffmpeg_${component}_LIBRARY")
 
